politeiawww/backend.go

```diff
@@ -1317,11 +1366,41 @@ func (b *backend) ProcessLogin(l www.Login) (*www.LoginReply, error) {
 	err = bcrypt.CompareHashAndPassword(user.HashedPassword,
 		[]byte(l.Password))
 	if err != nil {
+		user.FailedLoginAttempts += 1
+		if user.FailedLoginAttempts == LoginAttemptsToLockUser {
+			user.UserLocked = true
+			// Generate a new verification token and expiry.
+			token, expiry, err := b.generateVerificationTokenAndExpiry()
```

Comment:
I don't think we should reset the user's password in the email, we should just alert him that his account is locked and give him the url to reset his password.

If someone attacks a user's account and gets it locked while the user is on vacation or something, it's possible that the verification token generated here will just be expired when he checks his email again. So I think we should just alert the user and let him reset his password on his own time.